### ISSUE TITLE
Color Contrast Testing

### ISSUE THREAD

**Issue by Nancy-Salpepi** [phetsims/molecule-polarity#169]
Review of Screens 1 and 2:

Atoms
- labels on atoms have sufficient contrast with the atom color
- aside from label, atoms do not sufficient contrast with the background (light side, dark side and base color fails) 
- Atoms A and C: When overlapping, there isn't sufficient contrast between the 2 colors 
- Cueing arrows fail against background as well

Electronegativity and View panels 
- sufficient contrast between text and panel color
- black stroke around panel distinguishes it from background

Bonds
- sufficient contrast between bond color and background
- can't distinguish bonds when there is significant overlap

Electron potential and electron density gradients
- essential exception: color gradients that represent a measurement

Review of screen 3:

Electron potential and electron density gradients
- essential exception: color gradients that represent a measurement

Atom colors 
- essential exception: CPK coloring is used worldwide. These colors are fundamental to the information being conveyed and changing them would alter the meaning.

Molecular Polarity arrow
-black stroke was added to make it easier to see when going "through" an atom

**Comment by Nancy-Salpepi** [comment]
Added essential exceptions to WCAG Essential Exceptions doc. Closing.
